lib/phut/open_vswitch.rb

```diff
@@ -7,14 +7,17 @@ class OpenVswitch
       "#{Phut::ROOT}/vendor/openvswitch-1.2.2.trema1/utilities/ovs-ofctl"
 
     attr_reader :dpid
+    alias_method :datapath_id, :dpid
+    attr_reader :name
 
-    def initialize(dpid)
+    def initialize(dpid, name = nil)
       @dpid = dpid
+      @name = name || @dpid
     end
 
     def run
       fail "Open vSwitch (dpid = #{@dpid}) is already running!" if running?
-      system "sudo #{OPENFLOWD} #{options.join ' '}"
+      system("sudo #{OPENFLOWD} #{options.join ' '}")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.